stack/adding-a-new-package.rst

```diff
@@ -53,37 +53,49 @@ All LSST DM repositories on GitHub must be configured by a repository administra
 There are a number of settings required to ensure this and they are described below with URLs referring to the ``afw`` package.
 Replace ``afw`` with the relevant package name to get to the correct page on GitHub.
 
-1. On the main settings page for the repository, https://github.com/lsst/afw/settings, disable squash and rebase merging:
+1. On the main settings page for the repository, https://github.com/lsst/afw/settings, disable squash and rebase merging, and enable automatic deletion of head branches after merging a pull request:
 
-.. image:: /_static/build-ci/github_merge_button_settings.png
+.. image:: /_static/build-ci/github_pull_requests_settings.png
 
-2. Configure ``main`` branch to enable protections.
+.. note::
+
+  If the Settings tab is not visible at the top of the repo page, an administrator likely needs to grant admin privileges first.
+
+2. Configure the ``main`` branch to enable protections.
 For ``afw`` this is located at https://github.com/lsst/afw/settings/branches/ and can also be found from the "Branches" sidebar item on the settings screen.
 In the "Branch protection rules" section of that page you will have to click on "Add rule" to create a rule for ``main``.
-Add in ``main`` as the branch name pattern and then enable status checks and require that branches be up to date before merging.
-Administrators must be included in these protections since it's all too easy to make a mistake without realizing you have special override powers.
+First, add in ``main`` as the branch name pattern.
+Second, enable ``Require a pull request before merging``, but disable ``Require approvals``.
+Third, enable status checks and require that branches be up to date before merging.
+Finally, include Administrators in these protections, since it's all too easy to make a mistake without realizing you have special override powers.
 With checks enabled people will be able to use the GitHub merge button on Pull Requests and know that the :ref:`standard process <workflow-code-review-merge>` is being adhered to.
 
-.. image:: /_static/build-ci/github_branch_protection.png
+Once the above settings have been configured correctly, click CREATE to save the new rule.
+The new rule settings should look something like this:
+
+.. image:: /_static/build-ci/github_branch_protection_rule_settings.png
+
+To add this action, type ``lint`` into the search box, and select the ``lint`` GitHub action.
```

Comment:
Is this sentence in the right place?  It feels a bit odd in conjunction with the following note and given the overall flow of the steps that the package creator must go through.